packages/block-library/src/post-terms/index.php

```diff
@@ -26,7 +26,10 @@ function render_block_core_post_terms( $attributes, $content, $block ) {
 		return '';
 	}
 
-	$align_class_name = empty( $attributes['textAlign'] ) ? '' : ' ' . "has-text-align-{$attributes['textAlign']}";
+	$classes = 'taxonomy-' . $attributes['term'];
+	if ( isset( $attributes['textAlign'] ) ) {
+		$classes .= ' has-text-align-' . $attributes['textAlign'];
```

Comment:
Nitpick: I think it's better to use string interpolation as before: `"has-text-align-{$attributes['textAlign']}"`. Same goes for the above class name.